src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -83,11 +86,28 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
         this.setState({ isLoading: false });
     }
 
-    public onChange = (property: string, value: any) => {
+    private markFieldIsValid = (field: string, isValid: boolean) => {
```

Comment:
I think this needs renaming. Maybe refer to the state change it does so something like markInvalidFields or setInvalidFields?